TriblerGUI/widgets/trustpage/js/legend.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Draw the legend based on the current visible network.
+ * Handle clicks to show and hide the legend.
+ */
+
+/**
+ * Draw the legend based on the quantitative data of the graph currently visible
+ * @param data the data of the graph
+ */
+function drawLegend (data) {
+    var svg = d3.select("#legend");
+
+    // Link size
+
+    var linkSize = d3.scaleLinear()
+        .domain([data.min_transmission, data.max_transmission])
+        .range([config.link.strokeWidthMin, config.link.strokeWidthMax]);
+
+    svg.append("g")
+        .attr("class", "legendSizeLink")
+        .attr("fill", config.legend.textColor)
+        .attr("transform", "translate(10, 20)");
+
+    var legendSizeLink = d3.legendSize()
+        .classPrefix("linkSize")
+        .title("Data over connection")
+        .scale(linkSize)
+        .shape("line")
+        .labelWrap(70)
+        .shapeWidth(70)
+        .shapePadding(15)
+        .labelOffset(20)
+        .labels(function(i) {
+            return formatBytes(i.generatedLabels[i.i], 3);
```

Comment:
`i` is not really descriptive. Couldn't you name it so that the variable name is more descriptive?